packages/odyssey-react-mui/src/PasswordField.tsx

```diff
@@ -130,7 +132,11 @@ const PasswordField = forwardRef<HTMLInputElement, PasswordFieldProps>(
           endAdornment={
             <InputAdornment position="end">
               <IconButton
-                aria-label="toggle password visibility"
+                aria-label={
+                  inputType === "password"
+                    ? t("passwordfield.icon.label.show")
+                    : t("passwordfield.icon.label.hide")
```

Comment:
How can consumers override this translation?

Reply:
@shawyu-okta You can override all Odyssey text using `translationOverrides` on either `OdysseyProvider` or `OdysseyTranslationProvider`.

Reply:
@KevinGhadyani-Okta great! thank you.